python/lsst/cp/pipe/ptc.py

```diff
@@ -354,7 +354,7 @@ def runDataRef(self, dataRef, visitPairs):
         lookupTableArray = np.zeros((numberAmps, numberAduValues), dtype=np.float32)
 
         # Fit PTC and (non)linearity of signal vs time curve, produce linearizer
-        self.fitPtcAndNonLinearity(dataset, lookupTableArray, ptcFitType=self.config.ptcFitType)
+        dataset = self.fitPtcAndNonLinearity(dataset, lookupTableArray, ptcFitType=self.config.ptcFitType)
```

Comment:
It might be helpful to clarify the comment before this line. Given the comment “Fit..., produce linearizer”, I would expect this to return something called `linearizer`, not `dataset`.

Reply:
I have added a couple of extra comments in there.